src/components/helper/uc.helper.js

```diff
@@ -1,13 +1,19 @@
 const { Forbidden, AssertionError } = require('../../errors');
+const { equal: equalIds } = require('../../helper/compare').ObjectId;
 
 const trashBinResult = ({ scope, data, complete }) => ({ trashBinData: { scope, data }, complete });
 
 const validPermissionOperators = ['AND', 'OR'];
 
+const isSuperhero = (currentUser) => currentUser.roles.some((role) => role.name === 'superhero');
```

Comment:
I would prefer to give the superhero a permission like "DELETE_USER_FROM_ALL_SCHOOLS" and check for the permission instead of the role.
But it is OK like this.